frontend/src/components/Header.tsx

```diff
@@ -1,12 +1,19 @@
+import {useNavigate} from "react-router-dom";
 
 export default function Header() {
 
+    const navigate = useNavigate()
+
+    const routeToHome = () => {
+        navigate("/categories")
+    }
+
     return(
         <div>
             <div>Titel</div>
             <button>Login</button>
             <button>Logout</button>
-            <button>Home</button>
+            <button onClick={routeToHome}>Home</button>
```

Comment:
```suggestion
            <button onClick={() => routeToHome('categories')}>Home</button>
```

since you'll use this basic navigation for all buttons, consider working with a variable to reuse your navigate function and avoid code duplication.